### ISSUE TITLE
about the different dataset in the training

### ISSUE THREAD

**Issue by z-lan** [XUSean0118/DVSNet#11]
Hi,
When I train the network according to your prompt，I encountered the following problems.
1. When I took the first step in training, i.e.
![捕获](https://user-images.githubusercontent.com/45734673/57591244-fd64d780-7562-11e9-92a1-e9b5aaf0131e.PNG)
I found that the code used  video sequences  and   imagestrain_13_list.txt ，but there is another file named  imagestrain_10_list.txt .  
I want to know the difference between using these two files.   In addition, I wonder if this step must be trained using video sequences?
2. When I took the second step in training, i.e. 
![捕获](https://user-images.githubusercontent.com/45734673/57591508-5d0fb280-7564-11e9-88a7-95f73ddaae00.PNG)
Is the  train-data  the testcases  generated in the previous step?  
What is the  val-data-dir   path  and how to get the validation testcases？
I think  the testcases  generated in the previous step is val-data  because Y=confidence scores, but in  gentestcase.py  the   SAVE_DIR = './train/'.
How can I get these two data?

Looking forward to your help, thank you.

**Comment by XUSean0118** [comment]
Hi, @wobujiaoluoxaiohei 
1. In list dir is  imagestrain_13_list.txt for train decision network and imagesval_10_list.txt for validate decision network. And imagestrain_13_list contain image pairs in cityscapes leftImg8bit_sequence training set with max frame distance 13, imagesval_10_list contain image pairs in cityscapes leftImg8bit_sequence validation set with max frame distance 10.
Must use image pairs in video sequences to train decision network.

2. You can use the imagesval_10_list.txt in problem 1 to generate validation testcases and change SAVE_DIR = './val/'

**Comment by wobujiaoluoxaiohei** [comment]
Thank you very much for your detailed reply！
I'm very sorry to ask you a few more questions.  
In list dir  train_list.txt  contains image pairs in cityscapes  leftImg8bit_trainvaltest\leftImg8bit  training set  and  eval_list.txt  contains image pairs in cityscapes  leftImg8bit_trainvaltest\leftImg8bit  validation set.   
But I can't find any place to use these two files in the project . Where are they used in the training process?

**Comment by XUSean0118** [comment]
These two files are for training only segmentation network, not directly related to this repository.

**Comment by z-lan** [comment]
I see. Thank you very much for your kind help.
I sincerely wish you all the best.
